itest/supply_commit_test.go

```diff
@@ -0,0 +1,18 @@
+package itest
+
+import (
+	"github.com/lightninglabs/taproot-assets/taprpc/mintrpc"
+	"github.com/stretchr/testify/require"
+)
+
+func testPreCommitOutput(t *harnessTest) {
+	t.Log("Minting asset group with a single normal asset and " +
+		"universe/supply commitments enabled")
+	mintReq := issuableAssets[0]
+	mintReq.Asset.UniverseCommitments = true
```

Comment:
Before this would panic somewhere?

Reply:
I think it would just error during validation because the `GroupInfo` was expected to be set early, which was impossible when minting a new asset group.